tachyon/crypto/commitments/kzg/kzg_family_test.h

```diff
@@ -0,0 +1,87 @@
+#ifndef TACHYON_CRYPTO_COMMITMENTS_KZG_KZG_FAMILY_TEST_H_
+#define TACHYON_CRYPTO_COMMITMENTS_KZG_KZG_FAMILY_TEST_H_
+
+#include <stddef.h>
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "tachyon/crypto/commitments/kzg/kzg.h"
+#include "tachyon/crypto/commitments/test/bn254_kzg_polynomial_openings.h"
+#include "tachyon/crypto/transcripts/simple_transcript.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/bn254.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/g1.h"
+#include "tachyon/math/polynomials/univariate/univariate_evaluation_domain_factory.h"
+
+namespace tachyon::crypto {
+
+constexpr static size_t K = 4;
+constexpr static size_t N = size_t{1} << K;
+constexpr static size_t kMaxDegree = N - 1;
+
+template <typename PCS>
+class KZGFamilyTest : public testing::Test {
```

Comment:
can we do
```suggestion
class KZGFamilyTest : public FiniteFieldTest<math::bn254::BN254Curve>t {
```
& add `#include "tachyon/math/finite_fields/test/finite_field_test.h"` and delete line 33?

Reply:
Definitely It would work, but it isn't what I meant. I wanted the type to be only finite field type.